src/Land/Model/soil_bc.jl

```diff
@@ -156,14 +156,10 @@ function soil_boundary_flux!(
     _...,
 )
     bc_function = bc.scalar_flux_bc
-    # on faces at the top and bottom of the domain,
-    # the vector n̂ can point in either ± ẑ, depending on which side of the domain
-    # we are on. The user supplies a scalar flux F, such that the flux at the boundary
-    # is assumed to be F⃗ = F ẑ - i.e. the user doesn't need to worry about the normal vector.
-    # so, we take the absolute value of n̂ here.
-    # The minus sign is because the condition is applied on minus the flux.
-    # the same argument applies for the other directions.
-    diff⁺.soil.water.K∇h = abs.(n̂) * (-bc_function(aux⁻, t))
+    # The user supplies the magnitude of an inward flux, normal
+    # to the domain. Since n̂ points outwards, we would apply a minus sign
+    # However, the BC is on K∇h, which is minus the flux, so no sign is needed.
```

Comment:
hey @gagnelandmanna - I added this to the documentation...
https://clima.github.io/ClimateMachine.jl/previews/PR2199/APIs/Land/LandModel/#ClimateMachine.Land.Neumann

does this clear it up? Flux is a vector so it should have a magnitude and direction. With Neumann, we are assuming that the direction is inwards and normal to the surface, and therefore just pass in a scalar magnitude. Before we had done so by assuming F = f z^, and passing in f, now we are assuming F = -f n^, and passing in f. For total flexibility, we could design it so that the user passes in a vector directly, but normal flux BC are common.